style.css

```diff
@@ -950,6 +1155,134 @@ h4 {
     color: #1a2236;
   }
 
+  .modal-desktop {
+    position: absolute;
+    width: 100%;
+    height: 1711px;
+    left: 70px;
+    z-index: 2;
+    background: #fff;
+    box-shadow: 0 10px 20px rgba(0, 0, 0, 0.25);
+    justify-content: center;
+    align-items: center;
+    margin: auto;
+  }
```

Comment:
- [ ] As requested by the previous reviewer, you still need to work on your modal pop-up card as it is not well aligned to the center of the screen.
- [ ] I noticed after the modal pop-up has been displayed, then when you scroll down, the main page start showing, and the pop-up will be scrolled up. Kindly use fixed positioning to change this strange functionality.
- [ ] Also on the mobile version, the pop-up cards are not well aligned to the center and they are hitting the wall both to `left` and `right`. And I also notice that the card only pop-up at the top of the page when the `see project` is clicked on, it doesn't pop up as the screen is to be displayed on the window. Kindly peruse the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) to style your pop-up cards correctly